INSTRUMENTS/PHIDGET/PHIDGET22/PHIDGET22.py

```diff
@@ -9,14 +10,30 @@ def onVoltageRatioChange(self, voltageRatio):
     print("VoltageRatio [" + str(self.getChannel()) + "]: " + str(voltageRatio))
 
 
-@flojoy
-def PHIDGET22(dc_inputs: list[DataContainer], params: dict) -> DataContainer:
-    """Pressure Measurement with Phidget 22 sensors"""
+@flojoy(deps={"Phidget22": "1.14.20230331"})
+def PHIDGET22(
+    default: Optional[OrderedPair] = None,
+    n_sensors: int = 1,
+    calibration1: float = 0.015,
+    calibration2: float = 0.06,
+) -> OrderedPair:
+    """
+    The node Phidget allows you to record pressures from Flexiforce sensors using a Phidget InterfaceKit.
+
+    Parameters
+    -----------
+    n_sensors: int
+         Defines the number of pressure sensors connected to the Phidget Interface Kit.
+    calibration1: float
+    calibration2: float
+        Calibration parameters to convert voltage into pressure.
+    """
     voltage: list[float] = []
     pressions: list[float] = []
-    N = params["n_sensors"]
+    sensor_num: list[int] = []
 
-    for i in range(0, N):
+    for i in range(0, n_sensors):
+        sensor_num.append(i + 1)
         # Creation of an instance of the VoltageRationInput class
         voltageRatioInput = VoltageRatioInput()
```

Comment:
Python variables are typically named using snake_case. Therefore, it is recommended to rename this variable as voltage_ratio_input for consistency.